src/connection_wrap.h

```diff
@@ -0,0 +1,36 @@
+#ifndef SRC_CONNECTION_WRAP_H_
+#define SRC_CONNECTION_WRAP_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "async-wrap.h"
```

Comment:
I don’t think this `#include` is necessary? Compiling and linting pass for me without it.